### PR TITLE
Add new badges and tiers to dashboard

### PR DESCRIPTION
**Pull request by piazzatron**
### Description

This adds dashboard work for badges, as well as some UI on profile page & modal + drawer.

Still waiting on two assets from design:
- New animated video
- New copy for discord modal

### Dragons

Nah

### How Has This Been Tested?

Locally

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/piazz-dashboard-tiers

File: src/containers/user-badges/TierExplainerDrawer.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  const onClose = useCallback(() => {
+    dispatch(setVisibility({ modal: 'TiersExplainer', visible: false }))
+  }, [dispatch])
+
+  const tier = useProfileTier()
+
+  if (tier === 'none') return null
+
+  return (
+    <Drawer isOpen={isOpen} keyboardVisible={keyboardVisible} onClose={onClose}>
```
Comment (raymondjacobson) [inline src/containers/user-badges/TierExplainerDrawer.tsx:34]: how'd this guy end up working out?

File: src/containers/user-badges/utils.ts [hunk 2]
```diff
@@ -66,14 +66,19 @@ export const getWeiBalanceForUser = (
 export const makeGetTierAndVerifiedForUser = () =>
   createSelector(
     [getWeiBalanceForUser, getVerifiedForUser],
-    (wei, isVerified) => {
+    (
+      wei,
+      isVerified
+    ): { tier: BadgeTier; isVerified: boolean; tierNumber: number } => {
```
Comment (raymondjacobson) [inline src/containers/user-badges/utils.ts:72]: still feeling like multiple selectors is better :)

**Review by raymondjacobson — APPROVED** [review]

File: src/containers/artist-dashboard-page/Tiles.tsx [hunk 3]
```diff
@@ -145,69 +133,57 @@ export const WalletTile = ({ className }: { className?: string }) => {
...
       <>
-        <div className={styles.platformToken}>
-          <img
-            alt={'Platform Token'}
-            className={styles.platformTokenImage}
-            src={platformTokenImage}
+        <div className={styles.tokenHero}>
+          <video
+            autoPlay
+            src={spinnyVideo}
```
[the remark this answers is not in the document]
Comment (piazzatron) [reply src/containers/artist-dashboard-page/Tiles.tsx:166]: Ya can add a comment. This is rendered from a ~3D~ program (Cinema 4D), so we don't have access to the JSON lottie stuff you'd get from AE. Different pipline!

File: src/containers/user-badges/TierExplainerDrawer.tsx (hunk 1 above)
Comment (piazzatron) [reply src/containers/user-badges/TierExplainerDrawer.tsx:32]: excellent

File: src/containers/artist-dashboard-page/Tiers.tsx [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+import { useDispatch } from 'react-redux'
+import { setVisibility } from 'store/application/ui/modals/slice'
+import { pressDiscord } from 'store/token-dashboard/slice'
+
+const messages = {
+  title: '$AUDIO VIP TIERS',
+  subtitle1: `Unlock $AUDIO VIP Tiers by simply holding more $AUDIO!`,
+  subtitle2: `Advancing to a new tier will earn your profile a badge, visible throughout the app, and unlock various new features as they are released.`,
+  unlocks: 'UNLOCKS',
+  badgeType: (badge: string) => `✅ ${badge} Badge`,
```
[the remark this answers is not in the document]
Comment (piazzatron) [reply src/containers/artist-dashboard-page/Tiers.tsx:27]: done

**Pull request merged**